### DISCUSSION TITLE
face recognition for asian faces?

### DISCUSSION THREAD

**Discussion by xqyi** [photoprism/photoprism#4028] [Q&A]
I've noticed that the built-in models are trained on North American facial data, resulting in lower accuracy for Asian face recognition. In practical tests, the recognition performance is indeed not very satisfactory.

Is there a way for me to personally retrain the model to improve the recognition rate for Asian faces?

**Comment by zzzz0317** [comment]
Is it possible to allow users to train with photos from their own library?

**Comment by luposlip** [comment]
I have been experiencing the same. I think using a different and bigger (thus slower) model would be the solution. Just my two (euro) cents. But I don't know which to use and haven't investigated how to change it.

**Reply by luposlip** [reply]
It's a known issue. Hopefully a new model will enhance this soon!
https://docs.photoprism.app/user-guide/organize/people/#asian-faces-and-children

**Reply by ndfan77** [reply]
FWIW, face recognition hasn't been anything to write home about for "North American" faces either (in many cases not even finding a face that's clearly present -- which is worse because then you can't link at all).  There have been complaints, but for whatever reason the Photoprism maintainers always seem to find other things to work on.

**Comment by luposlip** [comment]
@lastzero I have essentials (bought a few days ago), if this can be fixed, I promise I'll buy Pro! :-)

**Reply by graciousgrey** [reply]
Improvements are planned :) You can find related issues here:

- https://github.com/photoprism/photoprism/issues/4669
- https://github.com/photoprism/photoprism/issues/1090

**Reply by lastzero** [reply]
You are welcome to test it again with these improvements:
- https://github.com/photoprism/photoprism/issues/5167

**Comment by ptr727** [comment]
Any suggestions for tuning parameters to use for better recognition?\
I'd rather merge than unmerge, and results with my family pictures that include many asian faces are just unusable.

**Reply by lastzero** [reply]
:point_right:  https://docs.photoprism.app/user-guide/ai/face-recognition/#tuning-tips

**Comment by luposlip** [comment]
FWIW, it works much better for me now (still not 100% there, but getting
close).
